### PR TITLE
os/selinux: s/realpath/readlink -f/

### PR DESCRIPTION
**Pull request by joshix**
- Change progression: readlink (corner failures) -> realpath (obsolescent) ->
readlink -f (desired canonicalization of symlink absolute path).
- Link the file-issue recomendation to the bugs/issues URL.
- Title Case Headings.
- Tune language/style.
- Prefix blocked cmd lines with prompt (`$`) to distinguish from cmd output.

### CODE REVIEW

**Comment by joshix** [comment]
@kayrus Addendum PR to your merge. I switched to `readlink -f`. Objections/PTAL?

File: os/selinux.md [hunk 1]
```diff
@@ -16,22 +20,25 @@ enable SELinux logging by running the following commands as root:
 * `semodule -DB`
 * `systemctl restart audit-rules`
```
Comment (kayrus) [inline os/selinux.md]: Should we add `$` here too?

**Comment by kayrus** [comment]
Added one note

File: os/selinux.md (hunk 1 above)
Comment (joshix) [inline os/selinux.md]: Yep. Good catch. Will fix squash merge shortly.

**Comment by joshix** [comment]
Actually will merge this, *then* fix the above comment and a few other bits.

**Pull request merged**